frontend/src/bundles/video_editor/components/timeline/timeline.tsx

```diff
@@ -0,0 +1,71 @@
+import { type DragEndEvent, type ItemDefinition, type Range, type ResizeEndEvent,type RowDefinition  } from 'dnd-timeline';
+import { TimelineContext  } from 'dnd-timeline';
+import { useCallback, useState } from 'react';
+
+import { TimelineView } from './subcomponents/timeline-view.js';
+
+interface TimelineProperties {
+	initialRange: Range;
+	initialRows: RowDefinition[];
+	initialItems: ItemDefinition[];
+  }
+
+  const Timeline: React.FC<TimelineProperties> = ({ initialRange, initialRows, initialItems }) => {
```

Comment:
This is a perfect example of a better approach (Destructuring)

Reply:
Following this structure